frame/contracts/src/weights.rs

```diff
@@ -166,2104 +166,2196 @@ pub struct SubstrateWeight<T>(PhantomData<T>);
 			// Standard Error: 0
 			.saturating_add(Weight::from_ref_time(2_000 as u64).saturating_mul(s as u64))
-			.saturating_add(T::DbWeight::get().reads(5 as u64))
-			.saturating_add(T::DbWeight::get().writes(6 as u64))
+			.saturating_add(T::DbWeight::get().reads(8 as u64))
+			.saturating_add(T::DbWeight::get().writes(9 as u64))
 	}
 	// Storage: Contracts CodeStorage (r:1 w:1)
 	// Storage: Contracts Nonce (r:1 w:1)
 	// Storage: Contracts ContractInfoOf (r:1 w:1)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: System Account (r:1 w:1)
 	// Storage: Contracts OwnerInfoOf (r:1 w:1)
+	// Storage: System EventTopics (r:2 w:2)
 	/// The range of component `s` is `[0, 1048576]`.
 	fn instantiate(s: u32, ) -> Weight {
-		Weight::from_ref_time(186_136_000 as u64)
+		Weight::from_ref_time(178_912_000 as u64)
 			// Standard Error: 0
 			.saturating_add(Weight::from_ref_time(2_000 as u64).saturating_mul(s as u64))
-			.saturating_add(T::DbWeight::get().reads(6 as u64))
-			.saturating_add(T::DbWeight::get().writes(5 as u64))
+			.saturating_add(T::DbWeight::get().reads(8 as u64))
+			.saturating_add(T::DbWeight::get().writes(7 as u64))
 	}
 	// Storage: Contracts ContractInfoOf (r:1 w:1)
 	// Storage: Contracts CodeStorage (r:1 w:0)
 	// Storage: Timestamp Now (r:1 w:0)
 	// Storage: System Account (r:1 w:1)
+	// Storage: System EventTopics (r:2 w:2)
 	fn call() -> Weight {
-		Weight::from_ref_time(149_232_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(4 as u64))
-			.saturating_add(T::DbWeight::get().writes(2 as u64))
+		Weight::from_ref_time(159_188_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(6 as u64))
+			.saturating_add(T::DbWeight::get().writes(4 as u64))
 	}
 	// Storage: Contracts CodeStorage (r:1 w:1)
+	// Storage: System EventTopics (r:1 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	// Storage: Contracts OwnerInfoOf (r:0 w:1)
 	/// The range of component `c` is `[0, 64226]`.
 	fn upload_code(c: u32, ) -> Weight {
-		Weight::from_ref_time(51_721_000 as u64)
+		Weight::from_ref_time(52_177_000 as u64)
 			// Standard Error: 0
-			.saturating_add(Weight::from_ref_time(48_000 as u64).saturating_mul(c as u64))
-			.saturating_add(T::DbWeight::get().reads(1 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+			.saturating_add(Weight::from_ref_time(47_000 as u64).saturating_mul(c as u64))
+			.saturating_add(T::DbWeight::get().reads(2 as u64))
+			.saturating_add(T::DbWeight::get().writes(4 as u64))
 	}
 	// Storage: Contracts OwnerInfoOf (r:1 w:1)
+	// Storage: System EventTopics (r:1 w:1)
 	// Storage: Contracts CodeStorage (r:0 w:1)
 	// Storage: Contracts PristineCode (r:0 w:1)
 	fn remove_code() -> Weight {
-		Weight::from_ref_time(30_016_000 as u64)
-			.saturating_add(T::DbWeight::get().reads(1 as u64))
-			.saturating_add(T::DbWeight::get().writes(3 as u64))
+		Weight::from_ref_time(37_011_000 as u64)
+			.saturating_add(T::DbWeight::get().reads(2 as u64))
```

Comment:
There are storage count changes, so the weights were definitely out-dated.  
[SWC compare](https://weights.tasty.limo/compare?repo=substrate&threshold=10&path_pattern=frame%2F*%2Fsrc%2Fweights.rs&method=guess-worst&ignore_errors=true&unit=weight&old=master&new=at/contract-weights).

Reply:
Yes. This was the PR that added topics to the event.